### PR TITLE
VZ-6196 Use vz to upgrade and uninstall in the multi-cluster pipeline

### PR DESCRIPTION
**Pull request by mgianatagh**
Use vz upgrade and vz uninstall in the multi-cluster pipeline

### CODE REVIEW

File: ci/multicluster/Jenkinsfile [hunk 1]
```diff
@@ -884,31 +842,16 @@ def upgradeVerrazzano() {
...
-                # Do the upgrade
-                echo "Following is the verrazzano CR file with the new version:"
-                cat ${v8oUpgradeFile}
-                kubectl apply -f ${v8oUpgradeFile}
-                # wait for the upgrade to complete
-                kubectl wait --timeout=30m --for=condition=UpgradeComplete verrazzano/my-verrazzano
-
-                # Get the install job(s) and mke sure the it matches pre-install.  If there is more than 1 job or the job changed, then it won't match
-                kubectl -n verrazzano-install get job -o yaml --selector=job-name=verrazzano-install-my-verrazzano > ${WORKSPACE}/verrazzano/platform-operator/scripts/install/build/logs/verrazzano-post-upgrade-job.out
+                # Download the operator.yaml for the target version that we are upgrading to
```
Comment (pshirodkar-orcl) [inline ci/multicluster/Jenkinsfile:849]: Download the yaml only once outside the for loop.

Comment (mgianatagh) [reply ci/multicluster/Jenkinsfile:849]: I can look at changing that.  The code previous to my change was already doing that and placing each download in the sub-tree for the cluster being upgraded.

**Review by pshirodkar-orcl — APPROVED** [review]

File: ci/multicluster/Jenkinsfile (hunk 1 above)
Comment (mgianatagh) [reply ci/multicluster/Jenkinsfile:849]: fixed

**Review by bobdonat — APPROVED** [review]

**Review by stevshap — APPROVED** [review]
LGTM

**Pull request merged**